examples/envied_example/pubspec.yaml

```diff
@@ -6,10 +6,16 @@ publish_to: none
 environment:
   sdk: ^3.0.0
 
+dependencies:
+  envied: ^1.0.0
+
 dev_dependencies:
-  build_runner: ^2.4.6
-  envied_generator: ^1.0.0
   lints: ^5.0.0
   test: ^1.24.9
-dependencies:
-  envied: ^1.0.0
+  build_runner: ^2.4.13
+  envied_generator: 
```

Comment:
I'm not sure why this is necessary. This is an example and should not really be run from the package itself. If desired, one can still create a pubspec_overrides.yaml file.